### PR TITLE
Fix getParameter return type declaration

### PR DESCRIPTION
**Pull request by MKietlinski**
Fixed return type declaration errors on extending SymfonyPage (#4)

### CODE REVIEW

File: src/Page/Page.php [hunk 1]
```diff
@@ -92,7 +92,7 @@ protected function verifyUrl(array $urlParameters = []): void
         }
     }
 
-    protected function getParameter(string $name): NodeElement
+    protected function getParameter(string $name): ?string
```
Comment (Zales0123) [inline src/Page/Page.php:95]: One thing, should it be `null` allowed indeed?

**Review by Zales0123 — APPROVED** [review]

File: src/Page/Page.php (hunk 1 above)
Comment (MKietlinski) [reply src/Page/Page.php:95]: So far i didn't need to get any other parameter than page url, which of course should't be null, but by default there are few other parameters that could (and in my case they are) be null like for example files path.

**Comment by loic425** [comment]
@Zales0123 I need this fix for this sylius pull request https://github.com/Sylius/Sylius/pull/9887

**Comment by Zales0123** [comment]
Thank you, Maciej!

**Comment by kochen** [comment]
thanks @MKietlinski.

Can we close the related issue #4 ?

**Pull request merged**